src/pages/RequestCallPage.js

```diff
@@ -174,10 +198,37 @@ class RequestCallPage extends Component {
         return {firstName, lastName};
     }
 
+    getWaitTimeMessageKey(minutes) {
+        if (!minutes) {
+            return 'requestCallPage.waitTime.calculating';
+        }
+        if (minutes > 300) {
+            // The wait time is longer than 5 hours, so just say that.
+            return 'requestCallPage.waitTime.fiveHoursPlus';
+        }
+
```

Comment:
The spacing is inconsistent. You have no space between the first two and then a space between the second two.

Reply:
Good catch!